src/index.html

```diff
@@ -9,6 +9,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo"
+        class="header__logo logo"
+      >
+
+      <nav class="header__navbar">
+        <ul class="header__menu menu">
+          <li class="header__item">
+            <a href="#" class="header__link is-active">
+              Apple
+            </a>
+          </li>
+          <li class="header__item">
+            <a href="#" class="header__link">
```

Comment:
```suggestion
            <a href="#samsung" class="header__link">
```
add more specific name for links